### PR TITLE
Non-alerting dashboards

### PR DESCRIPTION
**Pull request by CodeSandwich**
Adds missing dashboards to Grafana.

~Depends on https://github.com/radicle-dev/infra/pull/111, because requires ffnet to run fresh nodes~

### CODE REVIEW

File: registry/ffnet/grafana_alerts_dashboard.json [hunk 1]
```diff
@@ -643,6 +643,358 @@
...
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "substrate_best_block_length { kubernetes_cluster = \"ffnet\" }",
+          "interval": "",
+          "legendFormat": "",
```
Comment (geigerzaehler) [inline registry/ffnet/grafana_alerts_dashboard.json:690]: You can use `{{ instance }}` here for better labels

Comment (CodeSandwich) [reply registry/ffnet/grafana_alerts_dashboard.json:690]: The re-org counter tells how many reorganizations happened until now. It's increased by 1 whenever such event occurs, but never goes down. This way we won't miss anything that happened multiple times between Prometheus updates.

**Comment by CodeSandwich** [comment]
@geigerzaehler I've applied your comments. It seems that you're in the middle of the work on the dashboards and much broader changes. It doesn't make sense to merge the current state, I think that we should wait for your PR to document the right configuration and drop this one.

**Comment by geigerzaehler** [comment]
@CodeSandwich I reconciled this with the changes on on grafana. See the added commits for details. Can you have a look?

**Comment by CodeSandwich** [comment]
LGTM, are we adding the new dashboard JSONs to the infra repo?

**Comment by geigerzaehler** [comment]
> are we adding the new dashboard JSONs to the infra repo?

Yes, we should at some point.

**Pull request merged**